qulice-pmd/src/test/java/com/qulice/pmd/PmdValidatorTest.java

```diff
@@ -127,27 +127,26 @@ public void understandsMethodReferences() throws Exception {
                 "}"
             )
         );
-        try (final StringWriter writer = new StringWriter()) {
-            Logger.getRootLogger().addAppender(
-                new WriterAppender(new SimpleLayout(), writer)
-            );
-            final Validator validator = new PMDValidator();
-            boolean thrown = false;
-            try {
-                validator.validate(env);
-            } catch (final ValidationException ex) {
-                thrown = true;
-            }
-            MatcherAssert.assertThat(thrown, Matchers.is(Matchers.is(true)));
-            MatcherAssert.assertThat(
-                writer.toString(),
-                Matchers.not(Matchers.containsString("(UnusedPrivateMethod)"))
-            );
+        final StringWriter writer = new StringWriter();
```

Comment:
@mkordas it was merged from master commit hash `bfa9963185b3843f8720dd027ac5195e7da5b69f`. It is not my changes